PopcornEditor/src/butter.js

```diff
@@ -922,67 +922,127 @@ window.Butter = {
           finishedCallback( project );
         }
 
+        function initialMediaSources( qs ){
```

Comment:
(whereas project data can be sent in after 'loaded', as in the current demo; don't know if that would benefit from earlier injection)